### PR TITLE
Decor Random Mode

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### Reference Issues/PRs
#137 
#### What does this implement/fix? Explain your changes.
- `random` mode added to `decor` function

### CODE REVIEW

**Comment by sepandhaghighi** [comment]
@sadrasabouri Please review this PR 🥇

File: art/art.py [hunk 1]
```diff
@@ -382,6 +382,9 @@ def indirect_decoration(decoration):
     :return: decoration as str
     """
     decorations = DECORATION_NAMES
+    if decoration == "random" or decoration == "rand" or decoration == "rnd":
```
Comment (sadrasabouri) [inline art/art.py:385]: what about `decoration in ["random", "rand", "rnd"]`?

File: art/art.py [hunk 2]
```diff
@@ -552,24 +554,29 @@ def get_font_dic(font_name):
...
     Return given decoration part.
 
     :param  decoration: decoration's name
     :type decoration:str
     :param reverse: true if second tail of decoration wanted
     :type reverse:bool
-    :return decor's tail
+    :param both: both tails returning flag
+    :type bool: bool
+    :return: decor's tail or tails as str or list
```
Comment (sadrasabouri) [inline art/art.py:572]: anyway now I think it might be confusing.
what about `decor's tail as str or tails as list of str`?

File: art/art.py [hunk 3]
```diff
@@ -552,24 +554,29 @@ def get_font_dic(font_name):
...
     """
     if isinstance(decoration, str) is False:
         raise artError(DECORATION_TYPE_ERROR)
+    decoration = decoration.lower()
     decoration = indirect_decoration(decoration)
+    if both is True:
+        return DECORATIONS_MAP[decoration]
     if reverse is True:
         return DECORATIONS_MAP[decoration][-1]
     return DECORATIONS_MAP[decoration][0]
```
Comment (sadrasabouri) [inline art/art.py:582]: how should we handle `both=True, reverse=True`, here we are ignoring `reverse` but isn't it better if we raise a warning for this situation?

**Review by sadrasabouri — CHANGES_REQUESTED** [review]

File: art/art.py (hunk 1 above)
Comment (sepandhaghighi) [reply art/art.py:385]: You are right!
But I have a plan to solve this issue + some other minor issues in a separate PR (immediately after #138)

File: art/art.py (hunk 2 above)
Comment (sepandhaghighi) [reply art/art.py]: Solved in https://github.com/sepandhaghighi/art/pull/138/commits/4b8a4f4d138ed50bc150285db1351fffd3fe9abd

File: art/art.py (hunk 3 above)
Comment (sepandhaghighi) [reply art/art.py:582]: I think it's not a big deal.

**Review by sadrasabouri — APPROVED** [review]

**Pull request merged**